carsus/io/molecules/molecules.py

```diff
@@ -0,0 +1,225 @@
+import pandas as pd
+from carsus.io.util import read_from_buffer
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+BARKLEM_COLETT_DATA_URL = "https://raw.githubusercontent.com/tardis-sn/carsus-data-molecules-barklem2016/main/data/"
+
+
+class BarklemColett2016Reader(object):
+    """
+    A class for reading and parsing data from the Barklem & Collet 2016 dataset.
```

Comment:
The dataset which is for what? Molecules?

Reply:
Yup. The main feature is the equilibrium constants sampled on a grid of temperatures so you can do a quick and dirty molecular number density calculation. Hopefully the example notebook is clear enough, but I can add more to it.